lib/common/common.go

```diff
@@ -60,3 +63,21 @@ func ParseDockerURL(arg string) (*ParsedDockerURL, error) {
 	p, err := docker.ParseDockerURL(arg)
 	return (*ParsedDockerURL)(p), err
 }
+
+// IsValidLayerRefs validates a layer refs
```

Comment:
This was supposed to be a library-exposed helper, but I guess we may re-introduce it later as soon as any consumers need it.